harmonisation/specimen/basisOfRecord.md

```diff
@@ -0,0 +1,29 @@
+# Basis of Record
+
+## openDS
+Will be harmonised to `dwc:basisOfRecord`
+
+In general this field cannot be empty, as we use it to determine whether we should process the specimen or not.
```

Comment:
I'm not sure if this is your intention, but "in general" usually implies there are exceptions to the rule. It doesn't look like that's the case, though.

Reply:
@southeo Better be: in principle? To indicate empty fields are not allowed, or does it still have the same meaning then?

Reply:
In a digital specimen it is not a required field so yes it can be empty.